pkgs/development/python-modules/syndom/default.nix

```diff
@@ -0,0 +1,54 @@
+{ lib
```

Comment:
I'm not sure if this should be placed where listparser was and if it should be packaged differently since it's not *just* a Python library.

Reply:
If it is a library with python bindings it should be placed outside and converted to a python package